packages/Frontend/src/components/Authenticate.jsx

```diff
@@ -74,7 +91,10 @@ Authenticate.propTypes = {
   authenticate: PropTypes.func.isRequired,
   error: PropTypes.oneOfType([
     PropTypes.bool,
-    PropTypes.string,
+    PropTypes.shape({
```

Comment:
Always use PropTypes.shape instead of object, it's much clearer. Google always gives us back an object with a message and code when there's an error, very consistent and straight forward.